tests/common/plugins/sanity_check/checks.py

```diff
@@ -154,6 +155,78 @@ def check_dbmemory(dut):
     logger.info("Done checking database memory")
     return check_result
 
+def check_monit_services_status(check_result, monit_services_status):
+    """
+    @summary: Check whether each type of service which was monitored by Monit was in correct status or not.
+              If a service was in "Not monitored" status, sanity check will skip it since this service
+              was temporarily set to not be monitored by Monit.
+    """
+    check_result["services_status"] = {}
+    for service_name, service_info in monit_services_status.items():
+        check_result["services_status"].update({service_name: service_info["service_status"]})
+        if service_info["service_status"] == "Not monitored":
+            continue
+        if ((service_info["service_type"] == "Filesystem" and service_info["service_status"] != "Accessible")
+            or (service_info["service_type"] == "Process" and service_info["service_status"] != "Running")
+            or (service_info["service_type"] == "Program" and service_info["service_status"] != "Status ok")):
+            check_result["failed"] = True
+
+    return check_result
+
+def check_monit(dut):
+    """
+    @summary: Check whether the Monit is running and whether the services which were monitored by Monit are 
+              in the correct status or not.
+    """
+    logger.info("Checking status of each Monit entry...")
+    networking_uptime = dut.get_networking_uptime().seconds
+    timeout = max((MONIT_STABILIZE_MAX_TIME - networking_uptime), 0)
+    interval = 20
+    logger.info("networking_uptime = {} seconds, timeout = {} seconds, interval = {} seconds" \
+                .format(networking_uptime, timeout, interval))
+
+    check_result = {"failed": False, "check_item": "Monit services"}
```

Comment:
Here the `"check_item"` field needs to match with item name defined in constants.py, `"monit"`.

Reply:
Great suggestions! Fixed.

Reply:
@yozhao101 your change is incorrect. Xin called out the value you need to use: 'monit', and he also gave the reason: match value in constants.py. Please make change accordingly.

Reply:
Never mind, I misread the change. you made the change already.

Reply:
Can you please check again? I changed the value of field 'check_item' from `Monit services` to `monit` according to Xin's suggestion.